spec/unit/webmock_spec.rb

```diff
@@ -8,4 +8,12 @@
   it "should not require safe_yaml" do
     expect(defined?SafeYAML).to eq(nil)
   end
+
+  it "should alias enable_net_connect! to allow_net_connect!" do
+    expect(WebMock.enable_net_connect!).to eq(WebMock.allow_net_connect!)
```

Comment:
@owyongsk is that the right way to test that? is that not comparing the result of these two method invocations?

Reply:
Makes sense, I've changed it to using `Object.method`.